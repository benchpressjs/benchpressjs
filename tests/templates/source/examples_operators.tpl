
{{{ if !count }}}
Nobody here!
{{{ end }}}

{{{ if (name == password) }}}
Your password cannot be the same as your name!
Please try a different one.
{{{ end }}}

{{{ if (name != "Bob") }}}
Hello, visitor!
{{{ end }}}

