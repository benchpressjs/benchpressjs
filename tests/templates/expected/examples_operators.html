
Nobody here!

Your password cannot be the same as your name!
Please try a different one.

Hello, visitor!

That's an affirmative, Bob!

Hello, fellow human Jack. How was your day?

Dang, nothing here!

Hello, Jack the Fantastic!
